packages/contracts-bedrock/scripts/getting-started/config.sh

```diff
@@ -91,4 +92,4 @@ EOL
 )
 
 # Write the config file
-echo "$config" > deploy-config/getting-started.json
+echo "$config" > deploy-config/getting_started.json
```

Comment:
we should not commit any changes to "getting started" files as we have our own "regtest" ones.

Reply:
done